ambry-api/src/main/java/com/github/ambry/frontend/ContainerMetrics.java

```diff
@@ -50,6 +50,8 @@
   // 410
   private final Counter goneCount;
 
+  private final Counter totalQpsCount;
```

Comment:
Naming is weird, we don't need qps in the name, Counter is an implicit qps. We can just call it totalCount.

Reply:
Sure, updated.